.github/workflows/update-website.yml

```diff
@@ -0,0 +1,25 @@
+name: Update website
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - "integrations/integrations.json"
+
```

Comment:
We need a concurrency limit on this to ensure it’s serialized properly.

Reply:
Not sure if this is what we want, but I added this one:
```
concurrency:
  group: website-trigger-${{ github.ref }}
  cancel-in-progress: true
```